spec/API_specification/examples/tpch/q5.py

```diff
@@ -0,0 +1,58 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Any
+
+if TYPE_CHECKING:
+    from dataframe_api.typing import DataFrame, SupportsDataFrameAPI
```

Comment:
The `DataFrame` type doesn't seem to be used here?

```suggestion
    from dataframe_api.typing import SupportsDataFrameAPI
```